Carsus_atomic_data/Custom_atomic_data.ipynb

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
<h5>Make sure to use a spellchecker on your documentation. "quntities" is mispelled.</h5><p> </p><p>I'm not sure I'd make the distinction between elements and atoms, but also CARSUS can include molecular information and does not particularly include information on the quantities of chemicals in the model, so I'd just say explicitly "on the properties or atoms and molecules needed by STARDIS." </p><p> </p><p>Also, I'd remove the part about kurucz being recommended. Rather, I'd say that, for a detailed and accurate stellar spectrum we highly recommend looking into using a VALD linelist bundled into your atomic data. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>